apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.$projectParam._index/route.tsx

```diff
@@ -27,6 +27,49 @@ import {
   trimTrailingSlash,
 } from "~/utils/pathBuilder";
 import { BreadcrumbLink } from "~/components/navigation/NavBar";
+import { requireUserId } from "~/services/session.server";
+import { LoaderArgs } from "@remix-run/server-runtime";
+import invariant from "tiny-invariant";
+import { typedjson } from "remix-typedjson";
+import { analytics } from "~/services/analytics.server";
+import { JobsListPresenter } from "~/presenters/JobPresenter.server";
+
+export const loader = async ({ request, params }: LoaderArgs) => {
+  const { projectParam, jobId } = params;
+  const userId = await requireUserId(request);
+  invariant(projectParam, "projectParam not found");
+  invariant(jobId, "jobId not found");
```

Comment:
There is no jobId in the params in this route

Reply:
Yay! I figured it out! 

There shouldn't be a `jobId` param. 

Formerly it was something like this below: 

```tsx
export const loader = async ({ request, params }: LoaderArgs) => {
  const { projectParam, jobId } = params;
  const userId = await requireUserId(request);
  invariant(projectParam, "projectParam not found");
  invariant(jobId, "jobId not found");
```

But, I changed it to this:

```tsx
export const loader = async ({ request, params }: LoaderArgs) => {
  const { projectParam } = params;
  const userId = await requireUserId(request);
  invariant(projectParam, "projectParam not found");
  invariant(userId, "jobId not found");
```  
 Now the UI renders correctly.
  
  
![image](https://github.com/triggerdotdev/trigger.dev/assets/48127194/e2a941e8-68a7-424a-bcc8-b5306e109b84)

Reply:
Am I on the right track?